test/docker/docker_utilities.go

```diff
@@ -36,15 +36,11 @@ func GetDatabaseListings(t *testing.T, composeFile string, influxContainerName s
 	var bucketList []map[string]interface{}
 	var listing []string
 	err := json.Unmarshal([]byte(jsonData), &bucketList)
-	if err != nil {
-		fmt.Println("Error while decoding data ", err.Error())
-		return nil, err
-	}
+	require.NoError(t, err)
```

Comment:
My suggestion was to invoke `require.NoError(t, err)` in the caller.

This function already returns `..., error`, so we should continue surfacing the error and letting the caller handle it.